ebpf-io/ebpf.io-website
# feat: updates summit-2023 page

**Describe what changes this pull request brings**
This pull request updates summit-2023 page.


**Useful links:**
[Link to page design in Figma](https://www.figma.com/file/L49LK6NTH0A5PVwGSW25ub/EBPF-Summit?type=design&node-id=3400%3A4161&mode=dev)


**Steps to test:**
1. Open the [summit-2023 page](https://deploy-preview-477--ebpfio.netlify.app/summit-2023/)
1. Confirm that everything looks and works as expected
